frontend/packages/dev-console/src/components/monitoring/dashboard/__tests__/MonitoringDashboard.spec.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 import * as link from '@console/internal/components/utils';
-import MonitoringDashboard from '../MonitoringDashboard';
+import { MonitoringDashboard } from '../MonitoringDashboard';
 import ConnectedMonitoringDashboardGraph from '../MonitoringDashboardGraph';
 import { monitoringDashboardQueries, topWorkloadMetricsQueries } from '../../queries';
+import {
+  TimespanDropdown,
+  PollIntervalDropdown,
+} from '@console/internal/components/monitoring/dashboards';
```

Comment:
nit: can we move it up to line 4 after `import * as link from '@console/internal/components/utils';`